src/pages/my_page/user_profile/MyProjects.tsx

```diff
@@ -0,0 +1,62 @@
+import { List, TablePagination, Typography } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { CristinApiPath } from '../../../api/apiPaths';
+import { ListSkeleton } from '../../../components/ListSkeleton';
+import { RootState } from '../../../redux/store';
+import { SearchResponse } from '../../../types/common.types';
+import { CristinProject } from '../../../types/project.types';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../utils/constants';
+import { useFetch } from '../../../utils/hooks/useFetch';
+import { ProjectListItem } from '../../search/project_search/ProjectListItem';
+
+export const MyProjects = () => {
+  const { t } = useTranslation();
+
+  const user = useSelector((store: RootState) => store.user);
+  const userCristinId = user?.cristinId?.split('/').pop() ?? '';
+
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
+
+  const [projects, isLoadingProjects] = useFetch<SearchResponse<CristinProject>>({
+    url: userCristinId
+      ? `${CristinApiPath.Project}?query=.&page=${page + 1}&results=${rowsPerPage}&participant=${userCristinId}`
```

Comment:
Se om feilen er løst, slik at query kan fjernes før merging https://unit.atlassian.net/browse/NP-31378
```suggestion
      ? `${CristinApiPath.Project}?page=${page + 1}&results=${rowsPerPage}&participant=${userCristinId}`
```